.cspell.json

```diff
@@ -113,7 +113,7 @@
         "DTDs",
         "DUSTILOCK",
         "Dariusz",
-        "Dariusz)    -- megalinter/issues/2448",
+        "Dariusz) -- megalinter/issues/2448",
```

Comment:
Hmm... How come it was needed, it's kind of rare

Reply:
just restored it from origin, you can see a lot of similarities in cspell, do not know what is the purpose of it. 
![image](https://github.com/oxsecurity/megalinter/assets/3431813/f04af912-8021-4998-b596-63b56cfd7c25)
![image](https://github.com/oxsecurity/megalinter/assets/3431813/0ff2130e-d2f6-4d83-b923-dd51f28144ce)

Reply:
lets's try remove.

Reply:
Hmmm this issue was supposed to be solved :'(

Reply:
https://github.com/oxsecurity/megalinter/pull/2562 😢

Reply:
If someone here is a regex master, I'd appreciate some help :D

Reply:
@DariuszPorowski Was your branch started locally before that fix, just so we don't work uselessly?